scripts/projection_script.r

```diff
@@ -0,0 +1,65 @@
+setwd("/home/tec0/covid_booster_uptake")
+source("get_uptake_data.r")
+source("interpolate_dates.r")
+source("build_scaling_model.r")
+source("evaluate_scaling.r")
+source("build_projection_model.r")
+source("evaluate_projection.r")
+source("projection_functions.r")
+
+# Load 2022 IIS data for USA
+iis_usa_2022 <- get_uptake_data(
+  "https://data.cdc.gov/api/views/unsk-b7fc/rows.csv?accessType=DOWNLOAD",
+  state = "Location",
+  date = "Date",
+  cumulative = "Bivalent_Booster_18Plus_Pop_Pct",
+  state_key = "USA_Name_Key.csv",
+  date_format = "%m/%d/%Y",
+  start_date = "9/2/2022"
+)
+
+# Load 2022 NIS data for USA
+nis_usa_2022 <- get_uptake_data(
+  "https://data.cdc.gov/api/views/akkj-j5ru/rows.csv?accessType=DOWNLOAD",
+  state = "Geography",
+  date = c("Time.Period", "Year"),
+  cumulative = "Estimate....",
+  state_key = "USA_Name_Key.csv",
+  date_format = "%m/%d/%Y",
+  start_date = "9/2/2022",
```

Comment:
if *we* are manipulating dates, we always use https://en.wikipedia.org/wiki/ISO_8601, ie yyyy-mm-dd

the workflow should always be to take whatever the input date format is, and get it to ISO 8601 (or to date objects) as soon as possible

Reply:
oh, I see, you might be doing that below